tests/llama_stack/conftest.py

```diff
@@ -169,6 +180,9 @@ def test_with_remote_milvus(llama_stack_server_config):
     env_vars = []
     params = getattr(request, "param", {})
 
+    # Create the Postgres Deployment and Service
+    request.getfixturevalue(argname="postgres_service")
```

Comment:
If postgres_service is always used then it's better just to add `postgres_service` a parameter to this fixture, same as it is done for bucket_name